### PR TITLE
Regex tweaks, comment escaping

### PR DESCRIPTION
**Pull request by stefanor**

### CODE REVIEW

File: gengogettext.py [hunk 1]
```diff
@@ -276,6 +278,9 @@ def approve(job):
 
 def revise(job):
     comment = raw_input('Comment: ')
+    # Gengo's UI doesn't handle HTML in comments, correctly.
+    comment = cgi.escape(comment)
```
Comment (beck) [inline gengogettext.py]: lol

**Comment by beck** [comment]
:+1:

File: tests/test_gengogettext.py [hunk 2]
```diff
@@ -6,8 +10,16 @@
 import gengogettext
 
 
-class TestGengoGettext(unittest.TestCase):
+@contextlib.contextmanager
+def ignoring(exception, errno=None):
```
Comment (beck) [inline tests/test_gengogettext.py]: I'm guessing contextlib.supress is not good enough because it can't/doesn't check error nums?

Comment (stefanor) [inline tests/test_gengogettext.py]: And this isn't python 3

**Pull request merged**